File: python/bubble.py
def bubble(array):
    length = len(array)
    for i in range(length):
        for j in range(i + 1, length):
            if array[j] < array[i]:
                temp = array[i]
                array[i] = array[j]
                array[j] = temp
    return array

File: ts/bubble.ts
export function bubble(array: number[]) {
  for (let i = 0; i < array.length; i += 1) {
    for (let j = i + 1; j < array.length; j += 1) {
      if (array[j] < array[i]) {
        const temp: number = array[i];
        array[i] = array[j];
        array[j] = temp;
      }
    }
  }
  return array;
}

export function bubble2(array: number[]) {}
